source/_components/xiaomi_aqara.markdown

```diff
@@ -277,3 +277,15 @@ That means that Home Assistant is not getting any response from your Xiaomi gate
   - You should generate the key again using an Android Phone or alternatively an emulator such as [bluestacks](https://www.bluestacks.com). In some instances, there is an issue with keys being generated using the iOS application.
   - You need to make sure to have multicast support on your network. If you are running Home Assistant in a virtual machine (like Proxmox), try `echo 0 >/sys/class/net/vmbr0/bridge/multicast_snooping` on the host and restart the service or reboot the host.
 - If the required library "PyXiaomiGateway" cannot be installed you will need to install some missing system dependencies `python3-dev`, `libssl-dev`, `libffi-dev` manually (e.g., `$ sudo apt-get install python3-dev libssl-dev libffi-dev`).
+- If your gateway's MAC address starts with `04:CF:8C`, there is a good chance that the required port `9898` is closed on your gateway (you can check it with the Nmap utility, using the command `Nmap - sU {gateway_ip} -p 9898`). To fix that issue, you need to do these steps:
```

Comment:
The command is `nmap` not `Nmap`. Also, UDP scan requires root privileges.

Reply:
from my experience, it works also with `Nmap` (at least on macOS), but you are true, `sudo` is required. I will update the docs, thank you